File: Treino LOPR/while3.js
const prompt = require("prompt-sync")();

let i = +prompt("Quantos números você quer analisar");
while (i > 0) {
    let numero = Math.floor(Math.random() * 100);

    if(numero % 2 === 0){
        console.log(numero + " PAR" + "\n");
    }else{
        console.log(numero + " IMPAR" + "\n");
    }
    i--;
}

File: Treino LOPR/while3.py
import random

i = int(input("Quantos números você quer analisar"))

while i > 0 :
    numero = [random.randint(1, 100)]

    if numero % 2 == 0:
        print(numero, "PAR")
    else :
        print(numero, "IMPAR");
    i -= 1